modules/project_cleanup/function_source/main.go

```diff
@@ -235,6 +247,29 @@ func invoke(ctx context.Context) {
 		}
 	}
 
+	removeFirewallPolicies := func(folder string) {
+		logger.Printf("Try to remove Firewall Policies from folder [%s]", folder)
+		firewallPolicyList, err := firewallPoliciesService.List().ParentId(folder).Context(ctx).Do()
+		if err != nil {
+			logger.Printf("Fail to list Firewall Policies from folder [%s], error [%s]", folder, err.Error())
+			return
+		}
+		for _, policy := range firewallPolicyList.Items {
```

Comment:
We may have to handle pagination? I see a `NextPageToken`
https://pkg.go.dev/google.golang.org/api/compute/v1#FirewallPolicyList

Reply:
I think we don't need to handle it now because the default value for `maxResults` in a page is large for our context  (`500`)
https://pkg.go.dev/google.golang.org/api/compute/v1#FirewallPoliciesListCall.MaxResults

Reply:
Gotcha, yeah thats reasonable.